docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+
+### Cross-platform configuration
+
+Cross-platform configuration like the app icon, name, splash screen, and so on must be implemented manually in native code. These are often implemented very differently across platforms.
+
+**With Prebuild** cross-platform configuration is handled at the Config Plugin level and the user often just has to set a single value like `"icon": "./icon.png"` to have all icon generation taken care of.
+
+### Dependency side-effects
+
+Most complex native packages have additional setup required. For example, a camera library requires permission settings be added to the iOS `Info.plist` and the Android `AndroidManifest.xml` file. This additional setup can be considered a disjointed side-effect of a package. Pasting required side-effect code into your project's native files can lead to difficult native compilation errors. It's also hard for library authors to document every possible combination of code blocks to paste.
+
+**With Prebuild** library authors, who know how configure their library better than anyone, can create a testable and versioned script (Expo Config Plugin) to automate adding the required configuration side-effects. This means library side-effects can be more expressive, powerful, and stable. For native code side-effects, we also provide: [AppDelegate Subscribers](/modules/appdelegate-subscribers) and [Android Lifecycle Listeners](/modules/android-lifecycle-listeners) which come standard in the default [Prebuild template](#templates).
+
+### Orphaned code
+
+When you uninstall a package you have to be certain you removed all of the side-effects required to make that package work. If you miss anything, it leads to orphaned code that you cannot trace back to any particular package, this code builds up and makes your project harder to understand and maintain.
+
+**With Prebuild** the only side-effect is the Config Plugin entry in a project's Expo config (`app.json`), which will fail to evaluate when the corresponding package has been uninstalled, meaning it's nearly impossible to create orphaned configuration.
+
+## Anti Pitch
+
+Here are some reasons _Expo Prebuilding_ might **not** be the right fit for a particular project. There are also some notes on how we plan on alleviating the existing drawbacks eventually.
+
+### React Native versioning
+
+`npx expo prebuild` generates native code based on the version of `expo` a project has installed, so a project with SDK 46 (`expo@46.0.0`) would generate a `react-native@0.69.4` app.
+
+Expo has a quarterly release cycle, whereas `react-native` releases major versions at random intervals, with stability patches following a few weeks after. This means there are times where you cannot use `npx expo prebuild` with the latest release of React Native. This could potentially be circumvented by using a custom [Prebuild template](#templates), but this is not recommended.
```

Comment:
```suggestion
Expo releases a new version approximately every quarter, and `react-native` does not follow a calendar based release schedule. This means there are times where you cannot use `npx expo prebuild` with the latest release of React Native. This could potentially be circumvented by using a custom [prebuild template](#templates) if you are willing to experiment. You can also mitigate this by cherry-picking any changes you need from the latest version of React Native into a fork and using that in your project.
```